templates/navigation/links--oe-multilingual-content-language-block.html.twig

```diff
@@ -7,9 +7,8 @@
  */
 #}
 {% include '@oe_theme/compositions/ec-component-content-language-switcher/content-language-switcher.html.twig' with {
-  'expandable_id': expandable_id,
-  'unavailable': unavailable,
-  'current': current,
-  'options': options,
-  'icon_path': ecl_icon_path,
-} only %}
+  'expandable': expandable,
+  'message': message,
```

Comment:
Are we use this variable in composition. Can't find usage. I don't see or didn't notice.

Reply:
Yep, true, I'll fix this.